app-config.yaml

```diff
@@ -122,6 +122,13 @@ kafka:
         - localhost:9092
 
 integrations:
+  gcs:
```

Comment:
Added a comment below but I think we should name this `googleGcs` over `gcs` to keep the pattern of "provider" + "service" e.g. `awsS3`, `azureBlobStorage`. :)

Also, just want to keep naming consistency with existing TechDocs configs. https://github.com/backstage/backstage/blob/master/plugins/techdocs/config.d.ts

Reply:
Agreed!